influxql/parser_test.go

```diff
@@ -338,7 +337,7 @@ func TestParser_ParseStatement(t *testing.T) {
 
 		// CREATE USER statement
 		{
-			s: `CREATE USER testuser WITH PASSWORD pwd1337`,
+			s: `CREATE USER testuser WITH PASSWORD 'pwd1337'`,
```

Comment:
So we have to quote passwords now? I thought the parser would know to interpret whatever follows `PASSWORD` as a string. Am I mistaken?